core/contractHost.js

```diff
@@ -68,17 +71,18 @@ No invites left`;
     // the installation in descriptions rather than the source code
     // itself.
     install(contractSrc) {
+      contractSrc = `${contractSrc}`;
+      const contract = evaluate(contractSrc, {
```

Comment:
What are the consequences of moving `evaluate` to the `install` step rather than `spawn`? We can still spawn multiple copies of the same installation with different parameters, right?

Reply:
Yes.  

We needed to move contract = evaluate(...) so it would be accessible to installation.checkAmount(). Previously the only method in installation was spawn(), so it was okay that the contract was local state.